### ISSUE TITLE
Libre diffusion des données d'un JDD

### ISSUE THREAD

**Issue by jbrieuclp** [PnX-SI/GeoNature#1517]
Bon pour faire suite à #607, j'ouvre une issue dédiée à ce sujet :

J'ai pour ma part un énorme souhait sur la possibilité d'ouvrir l'accès en lecture (uniquement sur la synthèse et les métadonnées) à un JDD (ou pourquoi pas un CA) peu importe les rôles associés à ces métadonnées.

**Exemple :**
_J'appartiens à une structure "A" et on m'a assigné des droits de lecture R=2 sur geonature et ses modules.
Comme c'est le cas dans de nombreuses structures, des JDD de partenaires y sont présents et ma structure "A" n'est pas indiquée comme jouant un rôle dans ces JDD.
Alors même que ces données ont vocation à être diffusées (si on part du principe qu'elles sont publiques) avec mon droit d'accès R=2 je ne pourrais jamais avoir accès à ces données._

Du coup un truc génialissime serait d'avoir une checkbox au niveau du JDD pour dire si oui ou non le JDD peut être entièrement diffusé ou non sur la synthèse et les métadonnées.

Au niveau fonctionnel sur la synthèse et les métadonnées, si la case à cocher est activée, le CRUVED pour ce JDD ne serait pas exploité.
Pour les autres modules il n'y aurait pas de changement.

Pour répondre à Camille, je ne pense pas qu'il faille utiliser l'info qu'un JDD soit d'origine public ou non pour faire ça. Il peut exister des JDD d'origine privés (non financé par une autorité publique) qui peuvent être diffusés librement. 
Dans notre cas, au GRETIA, les données collectées lors d'AG ou de stages, qui sont des données d'origine privées, ont vocation à être diffusées ouvertement.

**Comment by camillemonchicourt** [comment]
Oui en effet, c'est pas forcément la notion de JDD public ou privé, mais ce que je voulais dire, c'est que cela doit certainement être dissocié du fonctionnement actuel d'association d'un acteur à un JDD, ça doit être une info plus globale comme tu l'indiques. :+1:

**Comment by DonovanMaillard** [comment]
L'Observatoire en Bretagne a mis en place un mécanisme de login automatique avec un profil public/public. 

En reprennant ce fonctionnement et en donnant les droits de lecture de ces JDD au profil public/public, ça peut peut-être répondre (au moins temporairement) au besoin ?

**Comment by DonovanMaillard** [comment]
se pose aussi pour nous une question assez proche d'avoir un champs "diffusion oui/non",non pas pour le rendre accessible sur la synthèse librement, mais au nmoins pour identifier ce qu'on doit envoyer ou non au SINP...

**Comment by jbrieuclp** [comment]
Ba heu... C'est pas ça Donovan que tu veux ?
https://github.com/PnX-SI/GeoNature/issues/1305#issuecomment-823143066

**Comment by camillemonchicourt** [comment]
Concernant l'accès libre à GeoNature cette fonctionnalité est disponible depuis la version 2.6 (ou 2.7 je sais plus), mais ce n'est pas vraiment en lien avec le sujet du besoin évoqué par Jean-Brieuc.

**Comment by DonovanMaillard** [comment]
ça permet d'ouvrir publiquement l'accès à ce que l'utilisateur public/public a droit... donc à priori, ça permet de répondre à une partie du besoin ?

**Comment by DonovanMaillard** [comment]
> Ba heu... C'est pas ça Donovan que tu veux ? [#1305 (comment)](https://github.com/PnX-SI/GeoNature/issues/1305#issuecomment-823143066)

Pas exactement, l'objectif n'est pas de définir un niveau de précision ou de dégrader l'information, simplement un booléen pour définir ce qui peut être diffusé publiquement ou non.

**Comment by jbrieuclp** [comment]
Oui, mais là le besoin est différent. 
L'idée n'est pas d'ouvrir l'accès à n'importe qui, je veux uniquement permettre au personne inscrite dans la base, donc des personnes identifiées d'avoir accès à certains JDD de données, pas tous.

**Comment by DonovanMaillard** [comment]
au temps pour moi! en effet, j'avais mal compris l'objectif...

**Comment by jbrieuclp** [comment]
> > Ba heu... C'est pas ça Donovan que tu veux ? [#1305 (comment)](https://github.com/PnX-SI/GeoNature/issues/1305#issuecomment-823143066)
> 
> Pas exactement, l'objectif n'est pas de définir un niveau de précision ou de dégrader l'information, simplement un booléen pour définir ce qui peut être diffusé publiquement ou non.

Si l'origine de la données est publique, tu dois la diffuser au plus fin niveau de précision existant. En revanche si la données est d'origine privée, tu peux permettre à l'utilisateur (au producteur du JDD) de spécifier son souhait de diffusion de ses données au SINP, ou en ce qui nous concerne, nos partenaires, tout-en lui permettant de choisir un niveau de diffusion souhaité.
Derrière cette info n'a aucun impact sur la synthèse. Je l'utilise uniquement sur les diffusions vers l'extérieure.

**Comment by DonovanMaillard** [comment]
Oui, dans notre cas ca serait donc toujours soit précis (même pour certaines données privées), soit pas de diffusion. Mais en effet, ca reste un même principe d'avoir des champs additionnels au niveau des métadonnées.

**Comment by camillemonchicourt** [comment]
Je ne sais pas si cela est vraiment inclut dans le travail de @jpm-cbna de pouvoir définir des JDD ouverts à tous les utilisateurs connectés, mêmes si ils ont un CRUVED R de 1 ou 2 sur la Synthèse.
A analyser si c'est existant ou pas.
Si non, il faudrait bien analyser comment permettre cela correctement.
